test/WebSites/FiltersWebSite/Controllers/ResourceFilterController.cs

```diff
@@ -31,7 +33,9 @@ private class ShortCircuitWithFormatterAttribute : Attribute, IResourceFilter
 
             public ShortCircuitWithFormatterAttribute()
             {
-                _formatters = new IOutputFormatter[] { new JsonOutputFormatter() };
+                _formatters = new IOutputFormatter[] { new JsonOutputFormatter(
+                    new JsonSerializerSettings(),
```

Comment:
I wasn't suggesting reusing the global settings in `JsonFormatterController`, just grabbing them as a starting point. In any case, the user is not mutating the settings here in `ResourceFilterController`. In `JsonFormatterController`, would need to either leave the code as I've changed or match whatever copying we choose in `NormalController` (above).

Separately the previous code in `JsonFormatterController` is fragile. Should not depend on the formatter's constructor to create a unique `JsonSerializerSettings` instance. Another good reason for #4409, killing the `SerializerSettings` property to death.